src/profile-logic/committed-ranges.js

```diff
@@ -164,10 +167,19 @@ export function stringifyCommittedRanges(
   return arrayValue.map(stringifyStartEnd).join('~');
 }
 
-export function getFormattedTimeLength(
+export function getFormattedLength(
   length: Milliseconds,
+  unit: string,
   precision: Milliseconds = Infinity
 ): string {
+  if (unit === 'bytes') {
+    return formatBytes(
+      length,
+      /*significantdigits*/ 2,
```

Comment:
Existing issue: Could you make the d uppercase so that it says `siginificantDigits`, here and below